integrations/google-ai.md

```diff
@@ -161,14 +162,14 @@ messages = [
     ChatMessage.from_user(content="What is the temperature in celsius in Berlin?")
 ]
 res = gemini_chat.run(messages=messages)
-weather = get_current_weather(**res["replies"][0].content)
+weather = get_current_weather(**json.loads(res["replies"][0].text))
```

Comment:
this change is related to a bug that we recently fixed (https://github.com/deepset-ai/haystack-core-integrations/pull/1241)